src/renderer/components/+namespaces/namespace.store.ts

```diff
@@ -5,7 +5,7 @@ import { Namespace, namespacesApi } from "../../api/endpoints/namespaces.api";
 import { createPageParam } from "../../navigation";
 import { apiManager } from "../../api/api-manager";
 
-const storage = createStorage<string[]>("context_namespaces", []);
+const storage = createStorage<string[]>("context_namespaces");
```

Comment:
If this were maintained in the active cluster it could persist across Lens restarts?

Reply:
For some reasons saving data to `localStorage` doesn't do actually work and after app restarts it's empty.
Probably need to investigate  `electronjs` docs / tweak some configuration for  [BrowserWindow](https://www.electronjs.org/docs/api/browser-window) or similar. 
@jakolehm any ideas?

Reply:
Does `localStorage` persist between quitting the final browser instance and starting a new one (I mean for a normal browser)?

Reply:
Tried this, doesn't work:

```
      this.mainWindow = new BrowserWindow({
        webPreferences: {
          partition: "persist:lens-app",
        },
      });
```

Probably we have to fallback to using plain json files as in `base-store.ts` (`conf` package).

Reply:
And remove all the instances of `createStorage`? Or create a new abstraction that syncs with both?

Reply:
> For some reasons saving data to _localStorage_ **doesn't work** and after app restarts _localStorage_ is empty.
> Probably need to investigate `electronjs` docs / tweak some configuration for [BrowserWindow](https://www.electronjs.org/docs/api/browser-window) or similar.
> @jakolehm any ideas?

It seems to persist if tray stays alive... so my guess is that localStorage is wiped because we get a new (random) port on boot.